extensions/gdx-tools/build.gradle

```diff
@@ -112,6 +112,6 @@ tasks.register('buildRunnables') {
 	}
 }
 
-tasks.named('processResources').configure {
-    dependsOn ':fetchNatives'
+processResources.configure {
```

Comment:
Does this really solve the underlying issue? 
If I remember correctly, there were way more issues with `fetchNatives`, because fetchNatives defines the whole project as a output.

Reply:
Well, I read the documentation linked in the error message. The last part mentions ordering as a possible solution, and that is what this does. You can try building it yourself. It's possible something was cached on my side, though I ran `gradlew clean` before any building task.

Reply:
Discord said earlier that you have to run those as separate lines.

`./gradlew fetchNatives`

`./gradlew build`

Reply:
Yes, I know, but then we also don't need the `mustRunAfter`, right?

Reply:
OK, the natives are also in tmp/ . I'm deleting that too now.

Reply:
> However, using what's in the PR, I just hand-deleted natives.zip, ran gradlew clean, and then ran gradlew build (without specifying fetchNatives at all).

That is expected, since `gradlew clean` doesn't remove extracted natives, as you can see e.g. under `gdx/libs`. So this should work with or without this pr.
As a side note, `mustRunAfter` does not create any dependency between `fetchNatives` and the `processResources` task.

You could build the project before this pr and the commit too, you just needed to run `fetchNatives` seperatly, which you still need to do after the commit/pr in most cases.

Reply:
OK, so we still need to get fetchNatives to run before anything else, and actually run... I'll try using both dependsOn and mustRunAfter, I guess. I can confirm that this can build but not run tests without fetching natives. Also, you can look at the GH Actions results to see that the build was actually failing, probably due to formatting issues.

Reply:
> OK, so we still need to get fetchNatives to run before anything else, and actually run...

I don't think I agree, `fetchNatives` should not be run automatically, otherwise you wouldn't be able to build natives locally

> you can look at the GH Actions results to see that the build was actually failing, probably due to formatting issues.

Yeah, but as you said, that was only formatting, I was only talking about the added dependsOn/mustRunAfter

Reply:
So now I'm much more confused. Why did @badlogic make [this, most-recent commit](https://github.com/libgdx/libgdx/commit/fd535ab38900cd266631a7bbeab83d10c1bc5403) in the first place? What was broken before? We know he didn't run spotlessApply, and that's why it won't build currently.

Reply:
That fix doesn't work, it should be reverted. As explained here https://github.com/libgdx/libgdx/pull/7246#issuecomment-1807271546 and as @Berstanio points out there are many other places where you would need to add a dependency on `fetchNatives` because it defines the whole project as an output.

Reply:
Then should this PR just revert the change in the most recent commit?

Reply:
I'd close this PR and let badlogic revert his commit.